src/types.ts

```diff
@@ -518,6 +520,7 @@ export type CustomMapper = (
     xmlParser: XMLParser
     registerReferencedItem: (identifier: string, path: NestedPath) => string
     buildPreviewId: (identifier: string) => string
+    buildMediaUrl: (url: string, rev?: number) => string
```

Comment:
Is this util usable from an outer scope? In that case I would suggest adding it somewhere in the docs. Out of the box it's hard to figure out what exactly this props should do.

Reply:
There is no documentation for the CaaSMapper at all. e-Spirit is in charge.